src/awsexplorer/activation.ts

```diff
@@ -209,3 +219,8 @@ function updateAwsExplorerWhenAwsContextCredentialsChange(
         })
     )
 }
+
+async function copyArnToClipboard(arn: string) {
+    await vscode.env.clipboard.writeText(arn)
+    vscode.window.setStatusBarMessage(localize('AWS.explorerNode.copiedArn', 'Copied ARN to clipboard'), 2000)
```

Comment:
I like this approach as a non-invasive, disappearing over time UX. Please move the 2000 into a constant.